fetch.bs

```diff
@@ -3023,6 +3027,75 @@ details of reused connections are not exposed and time values are coarsened.
 </div>
 
 
+<h3 id=preemptive-connections>Preemptive connections</h3>
+
+<div algorithm="preemmptively-obtain-a-connection">
```

Comment:
Does algorithm need this value? Normally it works without.

Reply:
Done